media/static-bundles.json

```diff
@@ -1107,6 +1119,12 @@
       ],
       "name": "firefox_whatsnew_91_de"
     },
+    {
+      "files": [
+        "js/firefox/whatsnew/up-to-date.js"
```

Comment:
This bundle should include the following scripts to make sure FxA flow params are attached to the subscription buttons:

```
"js/base/mozilla-fxa-product-button.js",
"js/base/mozilla-fxa-product-button-init.js"
```